publication-commons/src/main/java/no/unit/nva/publication/service/impl/TicketService.java

```diff
@@ -168,6 +167,19 @@ public TicketEntry updateTicketAssignee(TicketEntry ticketEntry, Username assign
         return updatedAssignee;
     }
 
+    public void removeTicket(TicketEntry ticketEntry) throws ApiGatewayException {
```

Comment:
I would not use `NotFoundException` as a catch-all exception here. `putItem()` may throw other types of exceptions. Handle failure with something like: `.orElseThrow(fail -> handlerTicketUpdateFailure(fail.getException()));` ?